### ISSUE TITLE
CheckedCRewriter unit tests fail after CConv changes

### ISSUE THREAD

**Issue by mgrang** [microsoft/checkedc-clang#851]
After https://github.com/microsoft/checkedc-clang/pull/837 merged we see the following three unit tests failing intermittently only on Windows:
```
- test/CheckedCRewriter/global.c
- test/CheckedCRewriter/simple_locals.c
- test/CheckedCRewriter/basic_checks.c
```
At least `test/CheckedCRewriter/global.c` seems to fail always when run manually on Windows.

```
$ "cconv-standalone" "clang\test\CheckedCRewriter\global.c" "--"
# command stderr:
Assertion failed: getPiece(StartPiece).size() > NumBytes, file clang\lib\Rewrite\RewriteRope.cpp, line 380
```

In order to unblock our failing ADO builds we need to disable the above three tests. We can enable them when we have these fixed.

**Comment by mgrang** [comment]
PR to disable tests: https://github.com/microsoft/checkedc-clang/pull/854

**Comment by mgrang** [comment]
I see that these tests have been enabled by the 3C team and they are passing. I will go ahead and close this bug as it is no longer relevant.
